commons/passive/fitbit/fitbit_activity_steps.avsc

```diff
@@ -0,0 +1,29 @@
+{
+  "namespace": "org.radarcns.passive.fitbit",
+  "type": "record",
+  "name": "FitbitIntradayHeartRate",
+  "doc": "Intra day heart rate data from fitbit device",
+    "fields": [
+        {
+            "name": "type",
+            "type": "string",
+            "doc": "Type of data coming from fitbit intraday API, (e.g. heart rate)"
```

Comment:
Is this always steps? In that case, this field can be removed. Additional schemas could be made to accommodate other data types. If this schema is meant to be more general, perhaps rename it to something like `FitbitIntradayMetric` instead. Even then, perhaps the data can differ based on the topic it was sent to, e.g. `fitbit_intraday_heart_rate` or `fitbit_intraday_steps`, and the type field would not be necessary.